source/unified-test-format/tests/valid-pass/initialCollectionData-collectionOptions.yml

```diff
@@ -0,0 +1,39 @@
+description: initialCollectionData-collectionOptions
+
+schemaVersion: '1.5'
+
+createEntities:
+  - client:
+      id: &client0 client0
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    collectionOptions:
+      capped: true
+      size: &cappedSize 512
+    documents:
+      - { _id: 1, x: 11 }
+
+tests:
+  - description: collection is created with the correct options
+    operations:
+      # Execute a collStats command to ensure the collection was created with the correct options.
+      - name: runCommand
+        object: *database0
+        arguments:
+          commandName: collStats
+          command:
+            collStats: *collection0Name
+            scale: 1
+        expectResult:
+          capped: true
+          maxSize: *cappedSize
```

Comment:
`collection is created with the correct options` fails in the Go driver on server versions 2.6 and 3.0. In 2.6, `maxSize` [is not returned](https://evergreen.mongodb.com/lobster/evergreen/test/mongo_go_driver_tests_legacy_auth_ssl__version~2.6_os_ssl_legacy~ubuntu1404_go_1_16_test_standalone_noauth_nossl_patch_0b0acdc5f38771ecf4d768179a315fe919d9f4b0_6259cfc42fbabe7dcb08d893_22_04_15_20_04_22/0/test/#bookmarks=0%2C39&l=1&shareLine=18) in the `collStats` command, and in `3.0`, `maxSize` [is 4096](https://evergreen.mongodb.com/lobster/evergreen/test/mongo_go_driver_tests_legacy_auth_ssl__version~3.0_os_ssl_legacy~ubuntu1404_go_1_16_test_standalone_noauth_nossl_patch_0b0acdc5f38771ecf4d768179a315fe919d9f4b0_6259cfc42fbabe7dcb08d893_22_04_15_20_04_22/0/test/#bookmarks=0%2C39&l=1&shareLine=18) (the default) instead of the expected 512. We could change this test to have a `minServerVersion: "3.6"` unless others are against.